Doc/library/importlib.rst

```diff
@@ -124,6 +124,11 @@ Functions
     need to call :func:`invalidate_caches` in order for the new module to be
     noticed by the import system.
 
+    If the module cannot be imported, :func:`import_module` will raise an
+    :exc:`ImportError`. If the module is found and loaded, but the code in the
+    module raises an exception, :func:`import_module` will pass that exception
+    to the module that called it.
```

Comment:
```suggestion
    If the module cannot be found or loaded, :func:`import_module` will raise an
    :exc:`ImportError`.
```

"cannot be imported" is very generic and could conceivably apply to explicitly raised exceptions, like in the second half of this paragraph, so makes sense to be more specific.

I think we could just omit the second half of the paragraph. Talking about "passing exceptions" makes it sound more complicated than it is. The behaviour here is intuitive and already documented by the sentence "This means all semantics of the function are derived from :func:`importlib.__import__`."

Reply:
Doesn't "cannot be found or *loaded*" imply that that's the behavior if the module cannot be loaded because code in the module raised an exception? Maybe "If the module cannot be found or its files cannot be accessed..." instead? That makes it clear that ImportError is raised in cases like the module not being found or being inaccessible due to file permissions, not cases where it is found but not successfully loaded.

Reply:
I suggest to look further down, where I tried to consolidate things. (sorry for the spam, I hope it's helpful for readers 😅)